terraform/.terraform.lock.hcl

```diff
@@ -0,0 +1,38 @@
+# This file is maintained automatically by "terraform init".
```

Comment:
we should remove this file and add its pattern to `.gitignore`

Reply:
Hi Dustin, that's one of the files I was debating putting into the `.gitignore` and then read through [this SO post](https://stackoverflow.com/questions/67963719/should-terraform-lock-hcl-be-included-in-the-gitignore-file) and kept it. But I'm happy to add it to the `.gitignore` if you'd like.

Reply:
interesting, i guess we'll keep it in for now 👍